evm/src/cpu/kernel/asm/sha2/memory.asm

```diff
@@ -0,0 +1,162 @@
+// Load a single byte from kernel general memory.
```

Comment:
@wborgeaud @dlubarov I realized this file isn't specific to Sha2 at all... should I just add it to `kernel/asm/memory/core.asm`?

Reply:
Yeah makes sense, let's have it near `mload_kernel_code`. Could also implement it as `%mload_kernel(@SEGMENT_KERNEL_GENERAL)`.

Reply:
Ohh okay! Good idea